test/RemoteExecutorConsoleApp/RemoteExecutorConsoleApp.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Runtime.ExceptionServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RemoteExecutorConsoleApp
+{
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            if (args.Length < 4)
+            {
+                Console.Error.WriteLine("Usage: {0} assemblyName typeName methodName exceptionFile [additionalArgs]", typeof(Program).GetTypeInfo().Assembly.GetName().Name);
+                Environment.Exit(-1);
```

Comment:
Why `Environment.Exit` and returning `-1`?

Reply:
I didn't change it due to reason mentioned here https://github.com/dotnet/machinelearning/pull/2200/files#diff-f3a47db0e2b37818c49fd22029730991R91